### PR TITLE
Add ECIP-1107 - Proof of Work Transition Process Testing

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

**Comment by bobsummerwill** [comment]
Thanks, @atoulme.

File: _specs/ecip-1105.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+ecip: 1105
```
Comment (q9f) [inline _specs/ecip-1105.md:2]:
```suggestion
ecip: 1107
```

**Comment by atoulme** [comment]
Comments addressed, please review.

**Review by gitr0n1n — APPROVED** [review]
LGTM for a WIP informational ECIP. Please continue to fill out the document, specifically the empty sections like backwards compatibility.

**Comment by gitr0n1n** [comment]
Not sure why the build is failing. Can you take a look @IstoraMandiri ?

**Review by gitr0n1n — CHANGES_REQUESTED** [review]
Figure out why this build is failing for this PR

File: _specs/ecip-1107.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+ecip: 1107
+title: WIP - Proof of Work Transition Process Testing
+lang: en
+translator: 
+author: Antoine Toulme (@atoulme)
+discussions-to: https://github.com/ethereumclassic/ECIPs/discussions/444
+status: WIP
```
Comment (meowsbits) [inline _specs/ecip-1107.md:8]:
```suggestion
status: Draft
```

Comment (gitr0n1n) [reply _specs/ecip-1107.md:8]: Interesting, so you think the build is failing due to the WIP designation? How do we submit an ECIP as a WIP then?

**Review by gitr0n1n — APPROVED** [review]
Look like the WIP status was preventing it from building like @meowsbits suggested! LGTM

**Pull request merged**